sample_shavar_list_creation.ini

```diff
@@ -97,31 +97,22 @@ output=fastblock2-trackwhite-digest256
 blocklist_url=https://raw.githubusercontent.com/mozilla-services/shavar-experiments/master/fastblock3.json
 output=fastblock3-track-digest256
 
-# DNT="", all top-level categories, fingerprinting tag
 [tracking-protection-base-fingerprinting]
-disconnect_tags=fingerprinting
-disconnect_categories=Advertising,Analytics,Social,Disconnect
+categories=Advertising|Analytics|Social|Content,Fingerprinting
```

Comment:
For example, here's the output I saw when I ran this ...

```
------ tracking-protection-base-fingerprinting ------
-->blocklist: https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/689b0f563f00c5ef55935c7248b357a57e588672/disconnect-blacklist.json)
 * filter ['Advertising', 'Analytics', 'Social', 'Content'] matched 2398 domains
 * filter ['Fingerprinting'] matched 39 domains. Reduced set to 29 items.
 * removing 1 rule(s) due to DNT exceptions
Tracking protection(tracking-protection-base-fingerprinting): publishing 30 items; file size 980

------ tracking-protection-content-fingerprinting ------
-->blocklist: https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/689b0f563f00c5ef55935c7248b357a57e588672/disconnect-blacklist.json)
 * filter ['Fingerprinting'] matched 39 domains
 * filter ['Advertising', 'Analytics', 'Social', 'Content'] matched 2398 domains
 * exclusion filters removed 29 domains from output
 * removing 1 rule(s) due to DNT exceptions
Tracking protection(tracking-protection-content-fingerprinting): publishing 11 items; file size 372
```

Is that right?

Reply:
You have the correct understanding of the filter syntax. If you see a better way to express it please let me know! That was the best I could come up with.

The absence of the Disconnect category is intentional. It can be added back in but the parser [silently drops it](https://github.com/mozilla/trackingprotection-tools/blob/4a1c0e1aea588e952264f5d0a98acdc075d36050/trackingprotection_tools/DisconnectParser.py#L376-L378) by default if it was [initialized with a remapping file](https://github.com/mozilla/trackingprotection-tools/blob/4a1c0e1aea588e952264f5d0a98acdc075d36050/trackingprotection_tools/DisconnectParser.py#L47-L49).

The way we were using the category previously was confusing. It was used more as a boolean flag to indicate whether or not to include remapped domains from the `Disconnect` category into the other categories specified by `disconnect_categories`. Take `disconnect_categories=Social,Disconnect` -- this wasn't used specify that we should keep all domains from the `Disconnect` and `Social` categories (as this would include non-`Social` domains for the `Disconnect` category). Rather, it meant to keep only those domains that map into the `Social` category from the `Disconnect` category. I thought about replacing it with a `remap_disconnect=True/False` config parameter which would then control whether to initialize the parser with the remapping file, but it's always true so I just hard coded it. I'm happy to add the `remap_disconnect` config option in and remove the `Disconnect` category from all of the new filters if you think it will help make it clearer. I can also just remove all of the `Disconnect` tags and make a note that the category is automatically remapped for all lists in a comment.

That is the expected output. I am unsure whether we should include the `Content` category domains in the `base` list, or restrict them to the `content` list (confusing naming of the lists on my part). Right now the split between the two lists is "fingerprinting trackers" and "fingerprinting non-trackers". If we move the Content category to the `content` list it becomes "fingerprinting trackers that we know don't break sites" and "fingerprinting trackers that we know break sites and fingerprinting non-trackers". I decided to go with the simpler version and see where we end up :).

Reply:
It seems like we could file a follow-up issue to re-name `Content` to `Strict` in the future? That seems to be the real way we're using the items in the `Content` category.

Reply:
Done in https://github.com/mozilla-services/shavar-list-creation/issues/75